src/eduid_userdb/user.py

```diff
@@ -69,8 +70,8 @@ def __init__(self, data, raise_on_unknown = True):
             _id = bson.ObjectId(_id)
         self._data['_id'] = _id
 
-        if 'sn' in self._data_in:
-            self._data_in['surname'] = self._data_in.pop('sn')
+        if 'surname' in self._data_in:
+            self._data_in['sn'] = self._data_in.pop('surname')
```

Comment:
Ok. There was some inconsistency producing errors, and I chose the SAML wording, I meant to ask you tomorrow about it. It was the same with mobiles and with entitlements. Will change it.